### ISSUE TITLE
Command fails with `which` commands for ubuntu 18.04

### ISSUE THREAD

**Issue by spacemonkey** [splash-cli/splash-cli#30]
### Basic informations
- **splash-cli** version: 3.0.2
- Node version: v8.12.0
- Your OS: Unbuntu 18.04 LTS

# ISSUE

Running `splash` with no arguments produces the following output:
```
✔ Connected!
✔ Making something awesome
Error: Command failed: which -a gsettings; which -a setroot; which -a pcmanfm; which -a feh; which -a nitrogen; which -a xfconf-query; which -a gconftool-2; which -a dcop; which -a dconf; which -a qdbus

    at ChildProcess.exithandler (child_process.js:276:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:915:16)
    at Socket.stream.socket.on (internal/child_process.js:336:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:561:12)
```

For grins I run the command by hand:
```
52% 2018-10-15 15:30:19 ⌚  monkeylappy in ~
○ → which -a gsettings; which -a setroot; which -a pcmanfm; which -a feh; which -a nitrogen; which -a xfconf-query; which -a gconftool-2; which -a dcop; which -a dconf; which -a qdbus
/usr/bin/gsettings
/usr/bin/dconf
```

**Comment by Rawnly** [comment]
Hi, can you try a clean installation of version 3.0.0? Just to see if it’s a module compatibility issue, unfortunately I can’t test on Ubuntu

**Comment by spacemonkey** [comment]
Ciao Fede! Looks like I'm getting an image returned (unlike 3.0.2), then crashing on the same command:

```
100% 2018-10-16 11:17:48 ⌚  monkeylappy in ~
○ → splash --version
3.0.0

100% 2018-10-16 11:17:52 ⌚  monkeylappy in ~
○ → splash


> silver spoon on bread

Downloaded: 2.7K times.
Viewed: 924.7K times.
Liked by 80 users.

Shot by: Monika Grabkowska (@moniqa)


Error: Command failed: which -a gsettings; which -a setroot; which -a pcmanfm; which -a feh; which -a nitrogen; which -a xfconf-query; which -a gconftool-2; which -a dcop; which -a dconf; which -a qdbus

    at ChildProcess.exithandler (child_process.js:276:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:915:16)
    at Socket.stream.socket.on (internal/child_process.js:336:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:561:12)
```

**Comment by Rawnly** [comment]
Maybe this could be a wallpaper issue.. because if it download an image the only thing that remains is set it as wp.

Please try run a script with the wallpaper >= 2.0.0

[the wallpaper module](https://github.com/sindresorhus/wallpaper)

**Comment by spacemonkey** [comment]
I updated to 3.0.2 and no more exit with error experience. The command error no longer happens, I have two new bugs to report so we should close this one out.

**Comment by Rawnly** [comment]
Ok awesome, let’s report these other 2 bugs so I can try to fix them in 3.0.3
